Specs/Scene/createOpenStreetMapImageryProviderSpec.js

```diff
@@ -10,9 +10,10 @@ defineSuite([
         'Scene/ImageryLayer',
         'Scene/ImageryProvider',
         'Scene/ImageryState',
-        'Specs/pollToPromise',
         'Scene/UrlTemplateImageryProvider',
-], function(
+        'Specs/pollToPromise',
+        
+    ], function(
```

Comment:
The formatting for this one got a little strange. remove the comma and whitespace.